apps/next/src/components/ui/nutrition-breakdown.tsx

```diff
@@ -0,0 +1,121 @@
+import { ProgressDonut } from "../calorie-progress-chart";
+
+// TODO: remove this variable and get the currently signed in user through session
+const DUMMY_USER_ID = "TEST_USER";
+
+interface NutritionData {
+  calories: number,
+  protein_g: number,
+  carbs_g: number,
+  fat_g: number
+}
+
+type LoggedMealJoinedWithNutrition = SelectLoggedMeal & {
+  calories: number,
+  protein: number,
+  carbs: number,
+  fat: number
+};
+
+function compileMealData(meals: (LoggedMealJoinedWithNutrition)[]): NutritionData {
+  const data = {
+    calories: 0,
+    protein_g: 0,
+    carbs_g: 0,
+    fat_g: 0
+  };
+
+  for (const meal of meals) {
+    const servings = meal.servings;
+    data.calories += meal.calories * servings;
+    data.protein_g += meal.protein * servings;
+    data.carbs_g += meal.carbs * servings;
+    data.fat_g += meal.fat * servings;
+  }
+
+  data.calories = Math.round(data.calories);
+  data.protein_g = Math.round(data.protein_g);
+  data.carbs_g = Math.round(data.carbs_g);
+  data.fat_g = Math.round(data.fat_g);
+
+  return data;
+}
+
+interface Props {
+  dateString: string,
+  mealsEaten: LoggedMealJoinedWithNutrition[]
+}
+
+const NutritionBreakdown = ({ dateString, mealsEaten }: Props) => {
+  const nutrition: NutritionData = compileMealData(mealsEaten);
+
+  const utils = trpc.useUtils();
+  const deleteLoggedMealMutation = trpc.nutrition.deleteLoggedMeal.useMutation({
+      onSuccess: () => {
+        alert(`Removed dish from your log`);
+        utils.nutrition.invalidate();
+      },
+      onError: (error: Error) => {
+        alert(error.message);
+      }
+    });
+
+  const removeBtnOnClick = (e: React.MouseEvent, userId: string | null, dishId: string | null) => {
+    e.preventDefault();
+    if (!userId || !dishId) return;
+
+    if (!DUMMY_USER_ID) {
+      alert("You must be logged in to track meals");
+      return;
+    }
+
+    deleteLoggedMealMutation.mutate({ userId, dishId });
+  };
+
+  return (
+    <div>
+      <center style={{ fontSize: '2rem', fontWeight: 'bold' }}>{dateString}</center>
+      <div className="flex align-items mt-4">
+        <div className="flex flex-col">
+          <center style={{ fontSize: '1.25rem', fontWeight: 'bold' }}>Calories</center>
+          <ProgressDonut progress_value={nutrition.calories} max_value={2000} display_unit="" />
+        </div>
+        <div className="flex flex-col">
+          <center style={{ fontSize: '1.25rem', fontWeight: 'bold' }}>Protein</center>
+          <ProgressDonut progress_value={nutrition.protein_g} max_value={75} display_unit="g" />
+        </div>
+        <div className="flex flex-col">
+          <center style={{ fontSize: '1.25rem', fontWeight: 'bold' }}>Carbs</center>
+          <ProgressDonut progress_value={nutrition.carbs_g} max_value={250} display_unit="g" />
+        </div>
+        <div className="flex flex-col">
+          <center style={{ fontSize: '1.25rem', fontWeight: 'bold' }}>Fat</center>
```

Comment:
Prefer Tailwind classes as they look better and are as efficient, if not more efficient than manually defining styles. 

```suggestion
      <center className="text-[2rem] font-bold">{dateString}</center>
      <div className="flex align-items mt-4">
        <div className="flex flex-col">
          <center className="text-[2rem] font-bold">Calories</center>
          <ProgressDonut progress_value={nutrition.calories} max_value={2000} display_unit="" />
        </div>
        <div className="flex flex-col">
          <center className="text-[2rem] font-bold">Protein</center>
          <ProgressDonut progress_value={nutrition.protein_g} max_value={75} display_unit="g" />
        </div>
        <div className="flex flex-col">
          <center className="text-[2rem] font-bold">Carbs</center>
          <ProgressDonut progress_value={nutrition.carbs_g} max_value={250} display_unit="g" />
        </div>
        <div className="flex flex-col">
          <center className="text-[2rem] font-bold">Fat</center>
```